src/main/java/Duke.java

```diff
@@ -14,15 +14,16 @@ public static void main(String[] args) {
             while(true) {
                 Scanner newScanner = new Scanner(System.in);
                 String user_input = newScanner.nextLine();
+                String[] user_input_split = user_input.split(" ");
 
-                if (user_input.equals("bye") | user_input.equals("Bye")) {
+                if (user_input_split[0].equals("bye") | user_input_split[0].equals("Bye")) {
                     System.out.println(" Bye. Hope to see you again soon!\n" +
                             "____________________________________________________________\n");
                     break;
                 }
 
-                else if (user_input.contains("mark")) {
-                    if (user_input.contains("unmark")) {
+                else if (user_input_split[0].contains("mark")) {
+                    if (user_input_split[0].contains("unmark")) {
                         char index = user_input.charAt(user_input.length()-1);
                         int indexint = Character.getNumericValue(index) - 1;
                         list[indexint].unmarkTask();
```

Comment:
in the lines that follow, extract the lines out as constants
line 33 'else' should be after the bracket